src/System.Private.ServiceModel/tools/scripts/WCFCustomSetup.cmd

```diff
@@ -0,0 +1,34 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+REM
+Echo Set Up test services
+REM
+
+REM ServiceUri is set, then the service is not local hosted
+if NOT "%ServiceUri%"=="" (
```

Comment:
I strongly recommend we do not require ServiceUri to be an environment variable.  It should be passed to this script from the override .targets file via $(ServiceUri).  Until we make ports configurable, it could just be %1 for now, but ultimately we probably want WcfCustomSetup.cmd /ServiceUri=XXX /HttpPort=YYY /HttpsPort=ZZZ, etc.